region/wreckedship/main.json

```diff
@@ -3084,6 +3192,43 @@
         {
           "from": 1,
           "to": [
+            {
+              "id": 1,
+              "strats": [
+                {
+                  "name": "Leave with Runway",
+                  "notable": false,
+                  "requires": [],
+                  "exitCondition": {
+                    "leaveWithRunway": {
+                      "length": 13,
+                      "openEnd": 0
+                    }
+                  }
+                }
+              ]
+            },
+            {
+              "id": 2,
+              "strats": [
+                {
+                  "name": "Come in with Runway, Leave Charged",
```

Comment:
Technically requires the item to be collected.

Reply:
We could add something like `itemCollectedAtNode`